serverless/interacting-serverless-apps.adoc

```diff
@@ -6,7 +6,11 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-This section provides information about various tasks that can be performed once you have successfully created a serverless application.
+This section provides information about various tasks that can be performed once you have successfully created a serverless application using Knative Services.
```

Comment:
Should `Services` be capitalized like this? The previous assembly has it lowered.

Reply:
Yeah it should be, I fixed some other instances so that it's consistent :slightly_smiling_face: